README.md

```diff
@@ -186,6 +186,66 @@ Update [values.yaml](https://github.com/openclarity/apiclarity/blob/master/chart
 
 4. Open APIClarity UI in the browser: <http://localhost:8080/>
 
+## Enabling External Trace Sources Support
+Enabling external trace sources support, APIClarity can receive the trace sources from the entitites that are external to the K8s cluster. External trace sources such as Gateways, Load balancers, etc. can communicate with APIClarity to report APIs and send the traces. 
+
+The following section describes how to deploy APIClarity with the support for external trace sources
+1)	Add Helm Repo
+```shell
+helm repo add apiclarity https://openclarity.github.io/apiclarity
+```
+
+2)	Update values.yaml with 
+```shell
+Apiclarity -> tls -> enabled as true
+supportExternalTraceSource -> enabled as true
+```
+
+3)	Deploy APIClarity with the Helm enabling external traffic sources
+```shell
+helm install --values values.yaml --create-namespace apiclarity apiclarity/apiclarity -n apiclarity
+```
+
+4)	Port forward to APIClarity UI:
+```shell
+kubectl port-forward -n apiclarity svc/apiclarity-apiclarity 9999:8080
+```
+
+5)	Open APIClarity UI in the browser:
+```shell
+http://localhost:9999 
+```
```

Comment:
I think this is already explained somewhere else in the README. Maybe it's better to make a reference to it rather than duplicating it here.

Reply:
Since the steps are different, will it be more helpful for the user with the inline steps? Thank you